ingest-document-blob-openai-cosmos/default/manifest.json

```diff
@@ -20,42 +20,42 @@
     },
     "parameters": [
         {
-            "name": "OpenAI_TextEmbedding_Deployment_#workflowname#",
+            "name": "OpenAI_TextEmbedding_Deployment",
             "displayName": "OpenAI text embedding deployment identifier",
             "type": "String",
             "description": "Provide the deployment model for Azure OpenAI.",
             "required": true
         },
         {
-            "name": "Blob_Path_pdf_ingestion_#workflowname#",
+            "name": "Blob_Path",
```

Comment:
I think _#workflowname# is required for all parameters.

Reply:
Here's the validation error related to this PR. We need to get them all fixed.

https://github.com/Azure/LogicAppsTemplates/actions/runs/14094176353/job/39477874483?pr=82